plugins/modules/system/lvol.py

```diff
@@ -389,7 +389,7 @@ def main():
     # Get information on volume group requested
     vgs_cmd = module.get_bin_path("vgs", required=True)
     rc, current_vgs, err = module.run_command(
-        "%s --noheadings --nosuffix -o vg_name,size,free,vg_extent_size --units %s --separator ';' %s" % (vgs_cmd, unit, vg))
+        "%s --noheadings --nosuffix -o vg_name,size,free,vg_extent_size --units %s --separator ';' %s" % (vgs_cmd, unit.lower(), vg))
```

Comment:
Why not to enhance this same piece of code until the end ?
```suggestion
        [vgs_cmd, "--noheadings", "--nosuffix",  "-o", "vg_name,size,free,vg_extent_size", "--units", unit.lower(), "--separator", ";", vg])

Reply:
What do you mean by this? As far as I see you only changed the syntax to use a list. 

I'm not familiar with coding ansible modules; Is this the recommended way of calling module.run_command?

Reply:
yes, it is: https://docs.ansible.com/ansible/latest/reference_appendices/module_utils.html
> args – is the command to run * If args is a list, the command will be run with shell=False. * If args is a string and use_unsafe_shell=False it will split args to a list and run with shell=False * If args is a string and use_unsafe_shell=True it runs with shell=True.

Reply:
It is, but let's not do too much in this bugfix and instead get it out faster. Such improvements can wait for afterwards.